trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/compute_source_coordinates.py

```diff
@@ -0,0 +1,272 @@
+## @ingroupMethods-Noise-Fidelity_One-Noise_Tools
+# compute_source_coordinates.py
+# 
+# Created: Mar 2021, M. Clarke 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ---------------------------------------------------------------------
+import numpy as np 
+from SUAVE.Core import Data
+# ----------------------------------------------------------------------
+#  Source Coordinates 
+# ---------------------------------------------------------------------
+
+## @ingroupMethods-Noise-Fidelity_One-Noise_Tools 
+def compute_point_source_coordinates(conditions,network,mls,source,settings):  
+    """This calculated the position vector from a point source to the observer 
+            
+    Assumptions:
+        N/A
+    Source:
+        N/A  
+    Inputs:  
+    conditions.
+          
+    network.
+    
+    mls                   - microphone locations       [m]
+    Outputs: 
+        position vector   - position vector of points  [m]
+    Properties Used:
+        N/A       
+    """  
+    
+    
+    # aquire dimension of matrix
+    num_cpt         = conditions._size
+    num_mic         = len(mls[0,:,0]) 
+    
+    if source == 'lift_rotors': 
```

Comment:
it's not that there is an instance of a particular converter (lift rotor or propeller), is a matter of determining what is the source. really haven't changed this since the last noise update, only updated the methodology of computing sound.

Reply:
I pulled the network out of the computation entirely, now there is not need for the instance()....